MonoGame.Framework/MacOS/MacGamePlatform.cs

```diff
@@ -380,6 +380,13 @@ public override void ResetElapsedTime()
         {
             _gameWindow.ResetElapsedTime();
         }
+		
+		public override void Present()
+        {
+            var device = Game.GraphicsDevice;
+            if (device != null)
+                device._graphicsMetrics = new GraphicsMetrics();

```

Comment:
I think all it means is that the Mac code didn't do it before... so changing it might break something.